python/python/lance/torch/kmeans.py

```diff
@@ -249,6 +249,10 @@ def _fit_once(
             del dists
             del chunk
 
+        if total_dist == 0:
+            logging.warning("Kmeans::train: total_dist is 0, this is unusual")
```

Comment:
In what condition can we get here?
Can you add comments for explanation?

cc @QianZhu @albertlockett